docs/solana/migration/hotspot-operator.mdx

```diff
@@ -99,14 +100,16 @@ billion tokens, distributed to all active Hotspots at the time of the Snapshot.
 The migration service will consider a Hotspot as "Active" if the Snapshot includes any of the
 following:
 
-- Witnessed and been valid/invalid witnesses.
-- Beaconed and had valid/invalid witnesses.
-- Transferred data.
+- Witnessed and was a valid witness
+- Beaconed and had valid witnesses
+- Transferred LoRaWAN data and been paid for it
 
 A Hotpsot is not "Active" if:
 
-- Beaconed with no valid/invalid witnesses, and
-- Has not passed data
+- Beaconed with no witnesses
```

Comment:
```suggestion
```
Beaconing with no witnesses is encapsulated in beaconing with no valid witnesses.